packages/ui-toolkit/src/label/index.js

```diff
@@ -10,7 +10,6 @@ const Label = styled.label`
   font-size: ${remcalc(15)};
   font-style: normal;
   font-stretch: normal;
-  font-weight: bold;
```

Comment:
I added here because all labels are bold in the design

Reply:
Yeah, Labels in the Forms. Labels in places like Service List they aren't. I only noticed now.

Notice that I added `bold` to FormLabel

Reply:
Ah! Awesome!